meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh

```diff
@@ -48,8 +49,7 @@ Options:
         Update supervisor getting the image from this registry.
 
   --supervisor-tag <SUPERVISOR TAG>
-        In the case of a successful host OS update, bring in a newer supervisor too
-        using this tag.
+        Before updating host OS, update resin supervisor using this tag.
```

Comment:
Perhaps let's be consistent in naming. README.md in resinhup calls these "ResinOS" and "Supervisor" (not "host OS" and "resin supervisor").